src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -190,28 +175,13 @@ class AddressValidationModal extends React.Component {
           className="vads-u-margin-top--2 vads-u-display--flex vads-u-align-items--center"
         >
           <div className="vads-u-display--flex vads-u-flex-direction--column vads-u-padding-bottom--0p5">
-            {addressLine1 && <span>{addressLine1}</span>}
-            {addressLine2 && <span>{` ${addressLine2}`}</span>}
-            {addressLine3 && <span>{` ${addressLine3}`}</span>}
-
-            {city &&
-              stateCode &&
-              zipCode && <span>{` ${city}, ${stateCode} ${zipCode}`}</span>}
-            {city &&
-              province &&
-              internationalPostalCode && (
-                <span>
-                  {`${city}, ${province}, ${internationalPostalCode}`}
-                </span>
-              )}
-            {/* State/Province/Region is not required with international addresses */}
-            {city &&
-              !province &&
-              internationalPostalCode && (
-                <span>{` ${city}, ${internationalPostalCode}`}</span>
-              )}
-
-            {displayCountryName && <span>{displayCountryName}</span>}
+            <div>
+              {street}
```

Comment:
I think we should stick with wrapping each of the three address elements in `span`s like we were doing before. Also if `country` isn't a string with a length > 0 we'll want to be sure to not render anything for it.

Reply:
I'll make sure to conditionally render the country! 👍 

For the `addressLines` however, we were rendering all 3 properties on one line on the Profile. The only place we were rendering them on spans on separate lines was in the address validation modal. I think we should stick with the same format in both places, and I chose to go with formatting on the Profile - I got approval from Matt to get the address to render on one line on the validation modal as well.

I could render separate spans, but then I'd need to explicitly return all separate lines from the `formatAddress` function, and then I'd also like to change how we render the address on the Profile for consistency. If it's up to me, I'd keep the current implementation 
 (rendering on one line) if you don't have a strong opinion on this :)

Reply:
sorry I meant wrap `street` `cityStateZip` and `country` each in their own span. Keeping one `street` is fine. Then you don't have to add the `br` tags (and won't have to worry about conditionally adding the `br` before `country` depending on if `country` is set). The fact that the container is a flex container means that the `span`s will be treated like block elements and stack on either other vertically.